src/content/configuration/dev-server.md

```diff
@@ -629,6 +629,16 @@ proxy: [{
 }]
 ```
 
+[Micromatch](https://www.npmjs.com/package/micromatch#matching-features) wildcard expressions are also supported:
+
+```js
+proxy: [{
+  context: ["/admin/**", "**/api/**", "(@/index.html/)"],
+  target: "http://localhost:3000",
+}]
+```
```

Comment:
You need to add another line break after the code block, before the next paragraph.